client/components/mma/cancel/cancellationSaves/SelectReason.tsx

```diff
@@ -194,12 +209,14 @@ export const SelectReason = () => {
 		mainPlan.chargedThrough ?? undefined,
 	).dateStr(DATE_FNS_LONG_OUTPUT_FORMAT);
 
+	const navigateToReminder = productType.productType === 'membership';
+
 	const submitReason = async () => {
 		{
 			const canContinue = !!selectedReasonId.length;
 			if (canContinue) {
 				await postReason();
-				navigate('../reminder', {
+				navigate(navigateToReminder ? '../reminder' : './', {
```

Comment:
```suggestion
				navigate(navigateToReminder ? '../reminder' : '/', {
```

this was the wrong path